### ISSUE TITLE
Flavor var file is used even if flavor option is not selected

### ISSUE THREAD

**Issue by Shilpi-Das1** [ocp-power-automation/openshift-install-power#155]
Getting error the following error while running script
Console Logs
```
Error: Failed to read variables file

Given variables file ./compute-vars/.tfvars does not exist.

```
**To Reproduce**
Command used to run ./openshift-install-powervs create -ignore-os-checks -verbose
where I am using default var.tfvars file

Expected result
It should take var.tfvars file as default and not use ./compute-vars/.tfvars if flavor option is not passed

**Comment by yussufsh** [comment]
@aishwaryabk please check this ASAP.

@Shilpi-Das1 if this is blocking you then please use the `master` branch or old version of the script till we get you the fix.

**Comment by Shilpi-Das1** [comment]
@yussufsh Sure I will use that I have tested it with PR https://github.com/ocp-power-automation/openshift-install-power/pull/153 Its running fine

**Comment by bpradipt** [comment]
Fixed via #153
